### ISSUE TITLE
Formatting to improve readability, link visibility in Docker paragraph

### ISSUE THREAD

**Issue by jessewashburn** [open-learning-exchange/open-learning-exchange.github.io#3406]
### Please verify the following:

- [X] This issue is related to the [Virtual Interns First Steps](https://open-learning-exchange.github.io/#!pages/vi/vi-first-steps.md) document or [Planet User Manual](https://open-learning-exchange.github.io/#!pages/manual/planet/overview.md).
- [X] You have searched for relevant keywords in [recent open and closed issues](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues?q=is%3Aissue+label%3Avi) to avoid duplicates.

### URL to the MDwiki page containing the problem

https://open-learning-exchange.github.io/#!./pages/robots/rbts-docker.md#What_is_Docker

### Problem

This section has a bit of a "wall-of-text" feel to it. Also, the Docker Linux Containers link could be made more visibile. Also, I'm not familiar with a "BeLL-App," but I'm asssuming those L's are capitalized on purpose. 

### Proposed solution(s)

Recommend breaking up the paragraph and inserting the link inside "Docker Linux Containers" See below:

However, to make this wiki coherent, we will talk a little bit about Docker. Docker is one of the implementations of container technology. Container technology implemented by Docker is the "younger brother" of virtual machine technology. With virtual machine technology, we can have a virtual computer inside a physical computer. And we can have more than one virtual machine if our physical machine is powerful enough. In different virtual machines inside a physical computer (say a server), we can install and setup applications like web application, network application, etc. This makes us easy to manage servers and make separation of some application in server easier. However, the main caveat of virtual machine is that it is resource hungry - meaning it takes a lot of processing power to do the virtualization. Although recent virtualization technology makes it more efficient, it still takes a lot of resources, and as you expect, another technology called container comes.   

The concept of container technology uses the paradigm of shipping containers in inter-modal transport. The idea is that before shipping containers were invented, manufacturers had to be prepared to ship goods in a wide variety of modes – ships, trains, or trucks – with different sized containers and packaging. By standardizing the shipping container, goods could be seamlessly transferred among shipping methods without any additional preparation. [Docker Linux Containers](https://www.sdxcentral.com/cloud/containers/definitions/what-are-containers-like-docker-linux-containers/) take the same approach with software. A container is a ready-to-use software package that can run in any operating system that supports it. For example, we can have a ready-to-use BeLL-App and when we want to install and run it in a new server, we can command the docker engine to run the container we have a working container. The software that packaged in a container share some part of the operating system of the host computer, but anything run in a container is isolated within a container.

**Comment by jessewashburn** [comment]
Apologies, this is not related to First Steps or Planet Manual, but rather, Contributors. I'll leave this here anyway though.

**Comment by RheuX** [comment]
Yea, thats an awfully long paragraph about docker, splitting it up into 2 or 3 is nice for the readers

**Comment by xyb994** [comment]
Yes, this is an older documents. Aside from its structure, the content is a bit outdated as well.

**Comment by ssamikshaa** [comment]
It's definitely a bit verbose and making it more concise and straightforward could help. I've also noticed that the BeLL app has been mentioned a few times in many of the pages. It might be beneficial to add either a page or a short explanation of what the BeLL app is to help users gain a better understanding of how it correlates to planet.
